basicswap/bin/prepare.py

```diff
@@ -929,16 +941,10 @@ def prepareCore(coin, version_data, settings, data_dir, extra_opts={}):
                 % (version, assert_filename)
             )
         elif coin == "namecoin":
-            release_url = "https://beta.namecoin.org/files/namecoin-core/namecoin-core-{}/{}".format(
-                version, release_filename
-            )
-            assert_filename = "{}-{}-{}-build.assert".format(
-                coin, os_name, version.rsplit(".", 1)[0]
-            )
-            assert_url = (
-                "https://raw.githubusercontent.com/namecoin/gitian.sigs/master/%s-%s/%s/%s"
-                % (version, os_dir_name, signing_key_name, assert_filename)
-            )
+            release_url = f"https://beta.namecoin.org/files/namecoin-core/namecoin-core-{version}/{release_filename}"
```

Comment:
Please use `https://www.namecoin.org/` rather than `https://beta.namecoin.org/`. Both domains serve content from the same directory on the server for the `/files/` subpath (their content only differs for Jekyll-generated content), but the `beta` domain has worse uptime because it's not intended for production usage (e.g. at some point we were testing some dodgy DNS records on the `beta` subdomain that made it go down for a few weeks).